### PR TITLE
Make preview size listbox invisible when Sizing = Fixed

### LINKED ISSUE #1571: Make preview size listbox invisible when Sizing = Fixed
Per recommendation by @jisqyv, it would be more intuitive if the preview dropdown didn't appear when Sizing is Fixed.

### PR DESCRIPTION
**Pull request by ZhengCornell**
Fixed #1571. Make drop-down list invisible when Sizing = Fixed, and make it visible when Sizing = Responsive. And the pre-setting will be preserved.

### CODE REVIEW

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java [hunk 1]
```diff
@@ -158,14 +159,13 @@ private void changeFormPreviewSize(int idx, int width, int height) {
     }
   }
 
-  public void enableTabletPreviewCheckBox(boolean enable){
+    public void enableTabletPreviewCheckBox(boolean enable){
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java]: Watch out for indentation here

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java [hunk 2]
```diff
@@ -933,8 +933,7 @@ public void onPropertyChange(String propertyName, String newValue) {
     } else if (propertyName.equals(PROPERTY_NAME_SIZING)) {
       if (newValue.equals("Fixed")){ // Disable Tablet Preview
         editor.getVisibleComponentsPanel().enableTabletPreviewCheckBox(false);
-      }
-      else {
+      } else {
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java]: is this edit necessary?

File: appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java [hunk 3]
```diff
@@ -133,6 +133,7 @@ public void onChange(ChangeEvent event) {
     });
     phoneScreen.add(listboxPhoneTablet);
 
+
```
Comment (josmas) [inline appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java]: be careful about these random edits! :)

**Comment by ewpatton** [comment]
I'm testing with Firefox 65.0.1 if that's of any help.

**Comment by ewpatton** [comment]
@ZhengCornell Now I have the opposite problem. If I create a project and set the Sizing to Responsive, it works until I reload the page. When the project reloads, it won't show the dropdown even though it should.

**Comment by ewpatton** [comment]
@ZhengCornell I have one last bug for you: If I switch to Responsive and set the size to be either tablet or monitor, and then switch from Response to Fixed, the size doesn't reset back to phone.

**Pull request merged**